doc/architectural/background-concepts.md

```diff
@@ -1218,23 +1218,24 @@ a map that associates both `fac` and `i` to "not 0".
 An abstract domain is a set $D$ (or, if you
 prefer, a data type) with the following properties:
 - There is a function merge that takes two elements of $D$ and returns
-  an element of $D$. This function is associative (merge(x, merge(y,z))
-  = merge(merge(x,y), z)), commutative (merge(x,y) = merge(y,x)) and
-  idempotent (merge(x,x) = x).
-- There is an element bottom of $D$ such that merge(x, bottom) = x.
+  an element of $D$. This function is:
+  * *associative* - `(merge(x, merge(y,z)) = merge(merge(x,y), z))`,
```

Comment:
If you want to highlight this as a keyword I’d prefer **bold** or none at all over cursive; Cursive is attention grabbing, but also often hard to read.

Very good idea to turn this into an actual list! I would also suggest removing the surrounding parentheses as they don’t really add anything, I think in the original sentence they were just there to delimit the expression which isn’t necessary in the list format anymore.